libadikted/lev_files.c

```diff
@@ -1289,8 +1289,20 @@ short write_lif(struct LEVEL *lvl,char *fname)
   message_log(" write_lif: starting");
   /*Acquiring map number */
   long lvl_num;
-  char *fname_num=fname;
-  while (((*fname_num)!='\0')&&(!isdigit(*fname_num))) fname_num++;
+
+  /**
```

Comment:
If a commit fixes an issue, no need to have leftover comments about the issue in the code. Explanations can go in the PR description, which can always be traced back by using 'git blame' on a line of code, if need be.

Reply:
Explanation removed.